server/views/pages/homepage.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "../partials/card.njk" import card %}
+
+{% set title = "Manage restricted patients" %}
+
+{% block pageTitle %}
+  Manage restricted patients
+{% endblock %}
+
+{% block main %}
+  <div class="govuk-width-container">
+    {{ govukBreadcrumbs({
+      collapseOnMobile: true,
+      items: [{
+        text: "Digital prison services",
+        href: pshUrl
+      }],
+      classes: "govuk-!-display-none-print"
+    }) }}
+
+    <h1 class="govuk-heading-l govuk-!-margin-top-7">{{ title }}</h1>
+  </div>
+
+  <div class="homepage-content govuk-!-margin-bottom-9">
```

Comment:
You'll need to add `_homepage .scss` from PSH to get the blue background for behind the tiles.